src/sections/references.html

```diff
@@ -0,0 +1,6 @@
+<section class="flex flex-col items-end pl-5 pr-10 my-10 bg-no-repeat bg-cover center sm:px-10 lg:px-40" style="background-image: url(./images/ref-yellow-shape.svg);">
```

Comment:
remove unnecessary spaces in your codebase
```suggestion
<section class="flex flex-col items-end pl-5 pr-10 my-10 bg-no-repeat bg-cover center sm:px-10 lg:px-40" style="background-image:url(./images/ref-yellow-shape.svg);">
```